packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -976,6 +976,17 @@ class WebpackCLI implements IWebpackCLI {
         description: "Stop watching when stdin stream has ended.",
         negatedDescription: "Do not stop watching when stdin stream has ended.",
       },
+      {
+        name: "fail-on-warnings",
+        configs: [
+          {
+            type: "enum",
+            values: [false],
+          },
+        ],
+        negative: true,
```

Comment:
This is misleading, it should be `false` if we don't want `--no-fail-on-warnings`. We actually prefer the `configs` property over this `negative` property so it will be ignored (like it was previously). So you can just remove it.

```suggestion
```